app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/jyou/y5/Y5Support.java

```diff
@@ -0,0 +1,570 @@
+package nodomain.freeyourgadget.gadgetbridge.service.devices.jyou.y5;
```

Comment:
This duplicates quite a lot of code of TeclastH30Support instead of reusing it. Why?